cdat_utilities/Masking_Data_Land-Sea_masks/Masking_Data_Land-Sea_masks.html

```diff
@@ -11766,73 +13108,165 @@
+.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
+.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
+.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
+.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
+.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
+.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
+.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
+.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
+.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
+.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
+</style></head>
+<body style=""><div id="MathJax_Message" style="display: none;"></div>
   <div tabindex="-1" id="notebook" class="border-box-sizing">
     <div class="container" id="notebook-container">
 
 <div class="cell border-box-sizing text_cell rendered"><div class="prompt input_prompt">
-</div>
-<div class="inner_cell">
+</div><div class="inner_cell">
 <div class="text_cell_render border-box-sizing rendered_html">
-<h1 id="Masking-Data-Land-Sea-Masks-">Masking Data Land-Sea Masks<a class="tocSkip" id="top"> </a><a class="anchor-link" href="#Masking-Data-Land-Sea-Masks-">&#182;</a></h1><p>In this example we will show how to generate masks, including land/sea masks</p>
+<h1 id="Masking-Data-Land-Sea-Masks-">Masking Data Land-Sea Masks<a class="tocSkip" id="top"> </a><a class="anchor-link" href="file:///home/nadeau1/Downloads/Masking_Data_Land-Sea_masks.html#Masking-Data-Land-Sea-Masks-">¶</a></h1><p>In this example we will show how to generate masks, including land/sea masks</p>
 <p>© The CDAT software was developed by LLNL. This tutorial was written by Charles Doutriaux. This work was performed under the auspices of the U.S. Department of Energy by Lawrence Livermore National Laboratory under Contract DE-AC52-07NA27344.</p>
-<p><a href="Masking_Data_Land-Sea_masks.ipynb">Download the Jupyter Notebook</a></p>
+<p><a href="file:///home/nadeau1/Downloads/Masking_Data_Land-Sea_masks.ipynb">Download the Jupyter Notebook</a></p>
```

Comment:
The notebook and all links within the html version of the notebook need to remain as relative links so the href="file:///home/nadeau1/Downloads/Masking_Data_Land-Sea_masks.ipynb" needs to remain href="Masking_Data_Land-Sea_masks.ipynb". All of the places in this file where "file:///home/nadeau1/Downloads/" exists needs to be removed.